postgresql-12.yaml

```diff
@@ -53,6 +53,40 @@ pipeline:
   - uses: strip
 
 subpackages:
+  - name: libecpg-12
+    description: PostgreSQL libraries
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib
+          mv ${{targets.destdir}}/usr/lib/libecpg.so* ${{targets.subpkgdir}}/usr/lib/
+    dependencies:
+      provider-priority: 12
```

Comment:
This feels like it's doing something clever, can you explain it in a comment?

Reply:
I copied it from a few other places in here: https://github.com/wolfi-dev/os/blob/main/postgresql-12.yaml#L81